lib/ekylibre/testing/application_controller_test_case.rb

```diff
@@ -0,0 +1,14 @@
+module Ekylibre
+  module Testing
+    class ApplicationControllerTestCase < ActionController::TestCase
+      include FactoryBot::Syntax::Methods
+      include Ekylibre::Testing::Concerns::FixturesCleanerModule
+
+      class WithFixtures < ActionController::TestCase
+        include FactoryBot::Syntax::Methods
```

Comment:
Can we extract this into a module instead of a class?
(and then `include WithFixtures` in the test cases instead of having a very long class name to inherit from)